data/json/items/armor/shields.json

```
@@ -84,5 +84,79 @@
     "environmental_protection": 2,
     "techniques": [ "WBLOCK_2" ],
     "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "BLOCK_WHILE_WORN" ]
+  },
+  {
+    "id": "shield_leather",
+    "type": "ARMOR",
+    "name": { "str": "leather reinforced shield" },
+    "description": "A simple round shield made of wood with leather reinforcement.  Sturdier and stronger than a simple wooden shield, and less encumbering than a metal one.",
+    "weight": "3200 g",
+    "volume": "3 L",
+    "price": "80 USD",
+    "price_postapoc": "8 USD",
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "wood" ],
+    "looks_like": "shield_wooden",
+    "symbol": "[",
+    "color": "blue",
+    "covers": [ "arm_either", "hand_either" ],
+    "coverage": 90,
+    "encumbrance": 16,
+    "material_thickness": 3,
+    "techniques": [ "WBLOCK_2" ],
+    "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "BLOCK_WHILE_WORN", "STURDY" ]
+  },
+  {
+    "id": "shield_banded",
+    "type": "ARMOR",
+    "name": { "str": "banded shield" },
+    "description": "A simple round shield made of wood with bands of steel reinforcing it.",
+    "weight": "3400 g",
+    "volume": "3 L",
+    "price": "100 USD",
+    "price_postapoc": "10 USD",
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "wood", "steel" ],
+    "looks_like": "shield_wooden",
+    "symbol": "[",
+    "color": "blue",
+    "covers": [ "arm_either", "hand_either" ],
+    "coverage": 90,
+    "encumbrance": 20,
+    "material_thickness": 3,
+    "techniques": [ "WBLOCK_2" ],
+    "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "BLOCK_WHILE_WORN", "NONCONDUCTIVE", "STURDY" ]
+  },
+  {
+    "id": "shield_leather_large",
+    "copy-from": "shield_leather",
+    "looks_like": "shield_wooden_large",
+    "type": "ARMOR",
+    "name": { "str": "large leather shield" },
+    "description": "A large shield made of wood with leather reinforcement.  Sturdier and stronger than a basic wooden shield, and less encumbering than a metal one.",
+    "volume": "5 L",
+    "weight": "5200 g",
+    "price": "80 USD",
+    "price_postapoc": "10 USD",
+    "encumbrance": 26,
+    "techniques": [ "WBLOCK_3" ],
+    "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "BLOCK_WHILE_WORN", "STURDY" ]
```

Can be simplified via using `extend`
```suggestion
    "extend": { "flags": [ "STURDY" ] },
```